live_view/live_view_studio/lib/live_view_studio_web/live/light_live.ex

```diff
@@ -0,0 +1,50 @@
+defmodule LiveViewStudioWeb.LightLive do
+  use LiveViewStudioWeb, :live_view
+  def mount(_params, _session, socket) do
+    socket = assign(socket, :brightness, 10)
```

Comment:
@samu975 what is a socket?

Reply:
a socket is a struct representing the WebSocket connection. In the socket struct, we stored the state of a live view process. In this case, we assign to the socket a brightness of 10